src/applications/gi-sandbox/containers/LocationSearchForm.jsx

```diff
@@ -54,6 +65,49 @@ export function LocationSearchForm({
 
   return (
     <div>
+      <div className="use-my-location-container">
+        {search.geolocationInProgress ? (
+          <div className="use-my-location-link">
+            <i
+              className="fa fa-spinner fa-spin"
+              aria-hidden="true"
+              role="presentation"
+            />
+            <span aria-live="assertive">Finding your location...</span>
```

Comment:
Why not use LoadingIndicator ?

Reply:
This is the way the facility locator has their indicator set up